core/_variables.scss

```diff
@@ -14,9 +14,13 @@ $base-line-height: 1.5;
 $heading-line-height: 1.2;
 
 // Other Sizes
+$space-xs: 0.5em;
```

Comment:
I’m all for it and have used that style on a few projects, [FormKeep being one of them](https://github.com/thoughtbot/formkeep/blob/master/app/assets/stylesheets/library/_variables.scss) (e.g. `$font-family--monospace`, `$color--body-text`).

But I do think this is a separate change/discussion outside of the scope of this PR, as we should change all the variables to a double-dash system if we want to go that route. Do you want to open a PR for this?